### PR TITLE
feat(platform): Add Publish agent flow section 

### PR DESCRIPTION
**Pull request by Bentlybro**
### Background

# Publish Agent Select
image
image

# Publish Agent Info
image
image

# Publish Agent Awaiting Review
image

### CODE REVIEW

File: autogpt_platform/frontend/src/components/agptui/PublishAgentAwaitingReview.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      <div className="w-full p-6 flex flex-col sm:flex-row items-center justify-center gap-4 border-t border-slate-200">
+        <button
+          onClick={onDone}
+          className="w-full sm:flex-1 h-12 flex items-center justify-center rounded-[59px] border border-neutral-900 bg-white"
+        >
+          <span className="font-['Geist'] text-sm font-medium leading-normal text-neutral-800">
+            Done
+          </span>
+        </button>
+        <button
```
Comment (Swiftyos) [inline autogpt_platform/frontend/src/components/agptui/PublishAgentAwaitingReview.tsx]: If possible can we use the agptui/Button component for buttons. 

I've come across cases though where we cant get the styling we need with it. But worth a shot

File: autogpt_platform/frontend/src/components/agptui/PublishAgentSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  return (
+    <div className="w-full max-w-[900px] bg-white rounded-3xl shadow-lg flex flex-col mx-auto">
+      <div className="p-4 sm:p-6 border-b border-slate-200 relative">
+        <div className="absolute top-4 right-4">
+          <button
+            onClick={onClose}
+            className="w-8 h-8 rounded-full bg-gray-100 flex items-center justify-center hover:bg-gray-200 transition-colors"
+            aria-label="Close"
+          >
+            <svg
```
Comment (Swiftyos) [inline autogpt_platform/frontend/src/components/agptui/PublishAgentSelect.tsx]: Can we add the svgs to components/ui/icons.tsx  in the same way the others have been. 

The is an icons.stories too so you can see what we have

File: autogpt_platform/frontend/src/components/agptui/PublishAgentAwaitingReview.tsx (hunk 1 above)
Comment (Bentlybro) [reply autogpt_platform/frontend/src/components/agptui/PublishAgentAwaitingReview.tsx]: Done this and it looks better :)

File: autogpt_platform/frontend/src/components/agptui/PublishAgentSelect.tsx (hunk 2 above)
Comment (Bentlybro) [reply autogpt_platform/frontend/src/components/agptui/PublishAgentSelect.tsx]: done this too

**Pull request merged**